### ISSUE TITLE
IOC log files: fix issue with garbled/misformatted output on restart with console command

### ISSUE THREAD

**Issue by FreddieAkeroyd** [ISISComputingGroup/IBEX#8593]
When an IOC is restarted via the console command, the output written in the log file starts to contain control characters and also wraps at 80 columns, this makes debugging more difficult. This does not happen on a restart if console is not attached, or if e.g. telnet is attached to the procServ instead of console.

## Acceptance criteria

- after patch, console to ioc and restart it using control-x and output should remain well formatted

## Notes

- this didn't used to happen, but started to happen at some point in the past

**Comment by FreddieAkeroyd** [comment]
So after a bit of digging around with terminal types and `forkpty` I discovered that this is due cygwin adding support for new windows pseudoterminals available in Windows 10 https://cygwin.com/pipermail/cygwin-developers/2019-April/011768.html, so the issue would have appeared when we moved to W10 and updated cygwin to a newer version. The new pseudoterminals are set at 80x25 initially and look like a real terminal, hence epics tries to use coloured output for errors. Previously it looked like a file and EPICS didn't use colour control characters. As not everything worked with the new API, cygwin later provided and option to disable this feature   https://cygwin.com/pipermail/cygwin-cvs/2020q1/014268.html
